Updated and modified some sections of troubleshooting docs

Roseline-Bassey opened the pull request:
Fixing #260  

This PR includes modifications and updates to the following sections in the troubleshooting docs:

1. [How do I troubleshoot issues with signatures?](  https://notaryproject.dev/docs/troubleshooting/#how-do-i-troubleshoot-issues-with-signatures) 

2. [Modified: "Enabling notation CLI commands logging"](https://notaryproject.dev/docs/troubleshooting/#enabling-notation-cli-commands-logging)

3. [ fixed missing closing parenthesis in this section](https://notaryproject.dev/docs/troubleshooting/#when-i-verify-an-artifact-i-get-the-error-error-open-homeconfignotationtrustpolicyjson-no-such-file-or-directory)

iamsamirzon commented on content/en/docs/troubleshooting.md:
```diff
@@ -11,92 +11,89 @@ To suggest a troubleshooting item, [create a pull request PR with your suggested
 This error indicates the trust store doesn't exist or the trust store name is not correct. Trust store typically contains a set of certificate files, where the trust identities are retrieved to verify signatures. You can use `notation cert add` to add trust stores.
 
-The above error shows the type of trust store is `ca`, and the trust store name is `mytruststore`. 
+The above error shows that the type of trust store is `ca`, and the trust store name is `mytruststore`. 
```
I think it will be good to callout that there are other types if trust stores as well, signingAuthority and tsa. We can even link to the spec - https://github.com/notaryproject/notaryproject/blob/5afbf9983756133dec85b7e8a323349fa10e3e56/specs/trust-store-trust-policy.md?plain=1#L52

Roseline-Bassey replied:
Thanks for pointing that out. I will make sure to include it.

iamsamirzon reviewed (approved):
Let's address that one comment below about calling out other types of trust stores : signingAuthority and tsa

Roseline-Bassey commented:
> @Roseline-Bassey Thanks. Could you resolve the DCO issue by signing your commits? See https://github.com/notaryproject/notaryproject.dev/pull/285/checks?check_run_id=14249077315

Thank you. I will address the issue.

Roseline-Bassey commented:
Hi, @FeynmanZhou, and @iamsamirzon 

I have addressed all issues in a new[ PR.](https://github.com/notaryproject/notaryproject.dev/pull/290). Please kindly take a look.

zr-msft commented:
@Roseline-Bassey I'm closing this PR based on this comment:
https://github.com/notaryproject/notaryproject.dev/pull/285#issuecomment-1595858695

Please reopen if its still relevant
